generator/AWSPSGeneratorLib/HelpMaterials/Examples/ELB2/Set-ELB2RulePriority.xml

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$rulePriority = New-Object Amazon.ElasticLoadBalancingV2.Model.RulePriorityPair
+$rulePriority.Priority = 11
+$rulePriority.RuleArn = 'arn:aws:elasticloadbalancing:us-east-1:123456789012:listener-rule/app/test-alb/3651b4394dd9a24f/a4eb199fa5046f80/dbf4c6dcef3ec6f8'
+
+Set-ELB2RulePriority -RulePriority $rulePriority -Region us-east-1
```

Comment:
Can be simplified to `-RulePriority @{Priority = 11; RuleArn = 'arn:aws:elasticloadbalancing:us-east-1:123456789012:listener-rule/app/test-alb/3651b4394dd9a24f/a4eb199fa5046f80/dbf4c6dcef3ec6f8'}`